src/test/java/org/fogbowcloud/manager/OrderStateTransitionerTest.java

```diff
@@ -34,7 +32,7 @@ private Order createOrder(OrderState orderState) {
         order.setOrderState(orderState);
```

Comment:
move this createOrder method to after the tearDown

Reply:
Done!